src/api/.haml-lint.yml

```diff
@@ -7,3 +7,9 @@ linters:
   InstanceVariables:
     exclude:
       - "app/views/**/*"
+
+  InlineStyles:
```

Comment:
regenerate the `.haml-lint_todo.yml` file instead of adding this in the `.haml-lint.yml` file. If you add it here, it is because you decided to disable the linter for any reason. If instead this is a linter that was not enable yet if should be in the todo file. And in this case doesn't make sense to do this manually because you have a generator. Just run (may happens that you need to delete the old one before):

`haml-lint --auto-gen-config`